test/release/examples/primers/fileIO.chpl

```diff
@@ -274,19 +274,22 @@ Error handling
 if example == 0 || example == 5 {
   writeln("Running Example 5");
 
-  // Error handling.
-  var err:syserr;
+  try! {
```

Comment:
It's not so clear to me that `try! { } catch e: SystemError { }` is the right pattern to show in these primers. Certainly it is closest to the previous behavior, but does it represent a "best practice"? I suppose it's unclear what we would do with another sort of error anyway... and that's when `try!` is useful.

Reply:
This was my thought, that `try!` says "I don't expect any other errors here".